ModernFlyouts/Controls/SessionControl.xaml.cs

```diff
@@ -21,12 +21,25 @@ public partial class SessionControl : UserControl
                 typeof(bool),
                 typeof(SessionControl),
                 new PropertyMetadata(true, OnAlignThumbnailToRightChanged));
+        
+        public static readonly DependencyProperty ShowThumbnailInBackgroundProperty =
+            DependencyProperty.Register(
+                nameof(ShowThumbnailInBackground),
+                typeof(bool),
+                typeof(SessionControl),
+                new PropertyMetadata(true, OnShowThumbnailInBackgroundChanged));
```

Comment:
These are not necessary. You could easily bind the **UIManager.ShowGSMTCThumbnailInBackground** property to the visibility of the border over here

https://github.com/ModernFlyouts-Community/ModernFlyouts/blob/a059480a40546b951b3396dce7da17cb624d4fea/ModernFlyouts/Controls/SessionControl.xaml#L25

Like this
```xaml
<UserControl ... xmlns:local="clr-namespace:ModernFlyouts" ...
....
<Border Grid.ColumnSpan="3" Grid.RowSpan="2"
    Opacity="0.5" CornerRadius="{DynamicResource FlyoutCornerRadius}"
    ClipToBounds="True"
    Visibility="{Binding Source={x:Static local:FlyoutHandler.Instance}, Path=UIManager.ShowGSMTCThumbnailInBackground, Converter={StaticResource BooleanToVisibilityConverter}}">
...
```